sentence.go

```diff
@@ -19,6 +19,66 @@ const (
 	ChecksumSep = "*"
 )
 
+var (
+	customParsers          = map[string]ParserCallback{}
+	defaultSentenceParsers = map[string]ParserCallback{
+		TypeRMC: func(s BaseSentence) (Sentence, error) {
+			return newRMC(s)
+		},
+		TypeGGA: func(s BaseSentence) (Sentence, error) {
+			return newGGA(s)
+		},
+		TypeGSA: func(s BaseSentence) (Sentence, error) {
+			return newGSA(s)
+		},
+		TypeGLL: func(s BaseSentence) (Sentence, error) {
+			return newGLL(s)
+		},
+		TypeVTG: func(s BaseSentence) (Sentence, error) {
+			return newVTG(s)
+		},
+		TypeZDA: func(s BaseSentence) (Sentence, error) {
+			return newZDA(s)
+		},
+		TypePGRME: func(s BaseSentence) (Sentence, error) {
+			return newPGRME(s)
+		},
+		TypeGSV: func(s BaseSentence) (Sentence, error) {
+			return newGSV(s)
+		},
+		TypeHDT: func(s BaseSentence) (Sentence, error) {
+			return newHDT(s)
+		},
+		TypeGNS: func(s BaseSentence) (Sentence, error) {
+			return newGNS(s)
+		},
+		TypeTHS: func(s BaseSentence) (Sentence, error) {
+			return newTHS(s)
+		},
+		TypeWPL: func(s BaseSentence) (Sentence, error) {
+			return newWPL(s)
+		},
+		TypeRTE: func(s BaseSentence) (Sentence, error) {
+			return newRTE(s)
+		},
+		TypeVHW: func(s BaseSentence) (Sentence, error) {
+			return newVHW(s)
+		},
+		TypeDBT: func(s BaseSentence) (Sentence, error) {
+			return newDBT(s)
+		},
+		TypeDBS: func(s BaseSentence) (Sentence, error) {
+			return newDBS(s)
+		},
+		TypeDPT: func(s BaseSentence) (Sentence, error) {
+			return newDPT(s)
+		},
+	}
+)
+
+// ParserCallback callback used to parse specific sentence variants
+type ParserCallback func(BaseSentence) (Sentence, error)
```

Comment:
The convention in Go is to use `Func` instead of `Callback` in types for functions. I'd prefer `ParseFunc`.